wp-content/themes/missilethreat/template-parts/content-actors.php

```diff
@@ -0,0 +1,145 @@
+        )
+  );
+  $missiles = get_posts( $args );
+}
+
+if ( $missiles ) : ?>
+
+	<div class="missile-table banded">
+	<h2><?php esc_html_e( 'Missiles', 'missiledefense' ); ?></h2>
+	<table id="missileTable">
+		<thead>
+			<th><?php esc_html_e( 'Missile Name', 'missiledefense' ); ?></th>
+			<th class="hidden-m"><?php esc_html_e( 'Class', 'missiledefense' ); ?></th>
+			<th class="hidden-m"><?php esc_html_e( 'Range', 'missiledefense' ); ?></th>
+			<th class="hidden-m"><?php esc_html_e( 'News', 'missiledefense' ); ?></th>
+		</thead>
+		<tbody>
+		<?php
+		/* Start the Loop */
+		foreach( $missiles as $post ) {
+			setup_postdata( $post );
+      $custom = get_post_custom();
+
+      $missileURL = get_field('missile_url');
+
+      if(isset($custom['missile_url'])) {
+        $url = esc_url(get_permalink($missileURL[0]));
+      }
+      else {
+        $url = esc_url( get_permalink());
+      }
+      
+      // Get the first part of the range
+      $range = null;
+      if(isset($custom['missile_range'])) {
+        // Check if we have a hyphenated range
+          if (strpos($custom['missile_range'][0], '-') !== false) {
+            $range = strtok($custom['missile_range'][0], '-');
+        }
+        else {
+          $range = strtok($custom['missile_range'][0], ' ');
+        }
+        $range = str_replace(",", "", $range);
+      }
+      
+      ?>
+      
+      <tr id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
+        <td>
+          <?php
+            if(isset($custom['missile_name']) && $custom['missile_name'][0] != '') {
+              if(get_post_status() != 'publish' && !isset($custom['missile_url'])) {
+                echo $custom['missile_name'][0];
+              }
+              else {
+                echo "<a href='".$url."' rel='bookmark'>".$custom['missile_name'][0]."</a>";
+              }
+            }
+            else {
+              if(get_post_status() != 'publish' && !isset($custom['missile_url'])) {
+                the_title();
+              }
+              else {
+                the_title( '<a href="'.$url.'" rel="bookmark">', '</a>');
+              }
+            }
+          ?>
+        </td>
+        <td class="hidden-m">
+          <?php
+            if(isset($custom['missile_class'])) {
+              echo $custom['missile_class'][0];
+            }
+          ?>
+        </td>
+        <td class="hidden-m" data-order="<?php echo $range; ?>">
+          <?php
+            if(isset($custom['missile_range'])) {
+              echo $custom['missile_range'][0];
+            }
+          ?>
+        </td>
+        <td class="hidden-m">
+          <?php
+            if(isset($custom['missile_status'])) {
+              echo $custom['missile_status'][0];
+            }
+          ?>
+        </td>
+      </tr>
+      <?php
+		}
+		?>
+		</tbody>
+	</table>
+	</div>
+
+
+  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.22/css/jquery.dataTables.min.css"/>
```

Comment:
Since we're redoing this, we should probably properly enqueue these styles & scripts. 😬  You can do that in `functions.php`, just make sure to add an if statement so it's only loaded on this page.